units/systemd-journal-remote.service.in

```diff
@@ -20,7 +20,7 @@ MemoryDenyWriteExecute=yes
 NoNewPrivileges=yes
 PrivateDevices=yes
 PrivateNetwork=yes
-PrivateTmp=yes
+PrivateTmp=disconnected
```

Comment:
this is wrong. The premise behind disconnected is that any quota established on `/tmp/` is taken away, along with the ability to inspect `/tmp/` of a service. We hence need to be very careful with enabling this, i.e. only when the dependency on `systemd-tmpfiles-setup.service` is strictly undesirable, but nothing else.

Please drop it.

Reply:
I mean, that's the whole deal for making `PrivateTmp=yes` backed by `/tmp/`, so that quota or setups where `/tmp/` is NOT a tmpfs is respected.

Reply:
Hmm, quota... OK. Dropped.